patchwork/models.py

```diff
@@ -823,6 +823,30 @@ class Meta:
         ]
 
 
+class Note(models.Model):
+    patch = models.ForeignKey(
+        Patch,
+        related_name='note',
+        related_query_name='note',
+        on_delete=models.CASCADE,
+    )
+    submitter = models.ForeignKey(Person, on_delete=models.CASCADE)
```

Comment:
This is a linked to a `Person`, but shouldn't it really be linked to a `User`? I expect only users can created notes? See `Check` for a similar example.

Reply:
I made submitter link to a `Person` following `EmailMixin` but considering notes are created through the Web UI/REST API it makes more sense to link to `User`. I'll swap it